index.html

```diff
@@ -5,8 +5,134 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <header class="page.header">
+      <img class="page-header-logo" src="" alt="Седона">
+      <nav class="navigation">
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link" href="main.html">Главная</a>
+            </li>
+            <li class="navigation-item">
+              <a class="navigation-link" href="about.html">О Седоне</a>
+              </li>
+              <li class="navigation-item">
+                <a class="navigation-link" href="hot.html">Гостиницы</a>
+                </li>
+                </ul>
+    <ul class="navigation-list navigation-user">
+      <li class="navigation-item">
+        <a class="navigation-link" href="#">
+          <span class="visually-hidden">
+            Поиск
+            </span>
+            </a>
+            </li>
+            <li class="navigation-item">
+              <a class="navigation-link" href="#">
+                <span class="visually-hidden">
+                  Избранное<sup>12</sup>
+                </span>
+              </a>
+            </li>
+            <li class="navigation-item">
+              <button class="navigation-item-button">Хочу сюда!</button>
+              </li>
+              </ul>
+              </nav>
+              </header>
+              </body>
+    <main class="main-index">
+      <h1 class="visually-hidden">Седона</h1>
+      <img class="logo" src="" alt="Логотип сайта «Седона».">
```

Comment:
не совсем понимаю, какой элемент должна представлять эта картинка, но это точно не лого. в первом блоке на главной странице можно выделить такие элементы 

![HTML 1 : Седона (Copy) 2022-01-23 20-54-41](https://user-images.githubusercontent.com/11165551/150691458-190fb15f-2f86-4795-9b8a-9389987e8366.jpg)

фоновая картинка будет добавлена позже через стили, декоративное изображение тоже. 
в разметке должно оказаться только векторное изображение с надписью. но помимо него должен быть блок, на который мы в стилях повесим фоновое и декоративное изображение. то есть можно сделать так 

```
<section class="hero">
    <h2 class="visually-hidden">Welcome to the gorgeous Sedona, because the Grand Canyon sucks!</h2>
    <img class="hero-image" src="" alt="декоративный текст">
</section>
```